src/main/edu/colostate/cs/cs414/method_men/jungle/client/Game.java

```diff
@@ -1,62 +1,69 @@
 package edu.colostate.cs.cs414.method_men.jungle.client;
 
-
+import java.util.Random;
 import java.util.Scanner;
 
 public class Game {
     private Player[] players;
     private int turn;
     private Board board;
+    private final int FAILURE = -1;
+    private final int SUCCESS = 100;
+    private final Location FAILURE_DESTINATION = new Location(FAILURE, FAILURE);
+    private final Location SUCCESS_DESTINATION = new Location(SUCCESS, SUCCESS);
 
     public Game () {
         players = new Player[2];
         players[0] = new Player("white");
         players[1] = new Player("black");
-        turn = 0; // white makes first move
+//        Random r = new Random(System.currentTimeMillis());
+//        turn = r.nextInt(999999) % 2;
+        turn = 0; // top Player makes first move
+        debugPrint(turn);
         board = new Board();
     }
 
+    /**
+     * Feel free to change the debug to true or false depending on if we're demo-ing or not
+     * @param message Could be an int, String, Object, doesn't matter it will print what you want
+     */
+    public void debugPrint(Object message) {
+        boolean debug = true;
```

Comment:
We probably do not need debug messages when we are not debugging.

```suggestion
        boolean debug = false;
```